packaging/macOS/launch_script.py

```diff
@@ -2,20 +2,14 @@
 
 import sys
 import os
-import locale
 
 from parsec.cli import cli
+from click import core
 
+# This prevents click from raising encoding errors at startup
+core._verify_python3_env = lambda: None
```

Comment:
It would be better to wrap the original `_verify_python3_env` so that we could log an error when it raises an exception instead of silence it. This would allow us to have more insight of when those errors occurs (OS version, locales used etc.)

However the crash occurs before we initialize sentry logging, so it would be a mess to catch the exception, save it on the side, then retrieve and log it after sentry initialization...
On top of that, we don't really care about unicode in click here (we are expecting to receive no arguments from argv or a single url which is should not contain unicode, similarly we output ascii logs to stdout but nobody read them anyway when starting the application this way)

So I think it's fine to just mock the way you did, but you may want to add my comment to the code in order to explicit why we consider this ok ;-)